src/fixed_width_ints.rs

```diff
@@ -92,5 +92,10 @@ cfg_if! {
 
         // static_assert_eq!(core::mem::size_of::<__uint128_t>(), _SIZE_128);
         // static_assert_eq!(core::mem::align_of::<__uint128_t>(), _ALIGN_128);
+    } else if #[cfg(all(target_arch = "aarch64", any(target_os = "macos", target_os = "ios", target_os = "tvos", target_os = "watchos")))] {
```

Comment:
Prefer using `target_vendor = "apple"` instead.

Reply:
I think I ve tried already in the past and now I try again I get this

```rust
  error: `cfg(target_vendor)` is experimental and subject to change (see issue #29718)
     --> ../src/fixed_width_ints.rs:95:50
      |
  95  |       } else if #[cfg(all(target_arch = "aarch64", target_vendor = "apple"))] {
      |                                                    ^^^^^^^^^^^^^^^^^^^^^^^
      | 

```

Reply:
Which job does this fail on? It looks like this should have been stable since 1.33 https://github.com/rust-lang/rust/pull/57465